src/crimson/os/seastore/onode.h

```diff
@@ -14,8 +14,14 @@
 namespace crimson::os::seastore {
 
 struct onode_layout_t {
+  // around 350 bytes for fixed fields in object_info_t,
+  // the left are for the variable-sized fields like oid
+  static constexpr int MAX_OI_LENGTH = 1024;
+
   ceph_le32 size{0};
```

Comment:
I don't think we need `onode_layout_t::size`, as it is already implemented in `Value::get_payload_size()`, see https://github.com/ceph/ceph/blob/master/src/crimson/os/seastore/onode_manager/staged-fltree/value.h#L180-L183

Reply:
That size isn't the onode size, it's the size of the object itself.

Reply:
...or will be shortly.